components/elements/table-body.vue

```diff
@@ -6,7 +6,8 @@
     <slot></slot>
     <div
       v-if="showEmpty"
-      class="col-span-1 px-6 text-sm py-4 grid grid-cols-1 md:grid-cols-5 text-gray-200 bg-gray-900 items-center rounded-sm"
+      class="col-span-1 px-6 text-sm py-4 text-gray-200 items-center rounded-sm"
+      :class="[light ? 'bg-gray-850 rounded' : 'bg-gray-900']"
```

Comment:
These ternary classes don't make much sense because it only allows for two viable options. We should utilise object-based classes, for example: 
```
:class="{'bg-gray-850 rounded': light, 'bg-gray-900': !light}"
```